### PR TITLE
[Method References] LPS-160434 Convert method references to field references in class extends BasePanelApp

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang ,I have fixed the bug about LegacyPortletPanelAppHotDeployListenerTest .testLegacyPortletWithControlPanelEntry.

Detial: https://github.com/liferay-core-infra/liferay-portal/pull/1798

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @zhangyan0701 , Could you please help me check the test results ? Thanks!

**Comment by W2jx** [comment]
ci:stop

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
Force-Push is just to rebase, rename portletName in LegacyPortletPanelAppHotDeployListenerTest and delete _testPortlet after the test

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/application/list/PublicationsPanelApp.java [hunk 1]
```diff
@@ -59,7 +64,6 @@ public PortletURL getPortletURL(HttpServletRequest httpServletRequest)
 		throws PortalException {
 
 		PortletURL portletURL = super.getPortletURL(httpServletRequest);
-
```
Comment (tinatian) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/application/list/PublicationsPanelApp.java:62]: @W2jx You should not remove this blank line

**Comment by W2jx** [comment]
Force-Push is just to rebase and fix the problem mentioned in comment.

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
Hi @zhangyan0701 , Could you please help me check the test results ? Thanks!

**Comment by zhangyan0701** [comment]
No related failures.

**Comment by tinatian** [comment]
@W2jx @dantewang Please see https://github.com/liferay-core-infra/liferay-portal/pull/1855/commits/9791cd54da0587652102a6fbce60f519b1684b73.

Tina

**Pull request closed without merging**